src/library_html5_webgl.js

```diff
@@ -78,10 +78,11 @@ var LibraryHtml5WebGL = {
   '$JSEvents', '$emscripten_webgl_power_preferences', '$findEventTarget', '$findCanvasEventTarget'],
   // This function performs proxying manually, depending on the style of context that is to be created.
   emscripten_webgl_do_create_context: function(target, attributes) {
+    {{{ convertPtrToIdx('target') }}};
```

Comment:
Again, because `emscripten_webgl_do_create_context` has it signature set in `library_sigs.js` to `ipp` these incoming pointers should already have been converted (at least in the wasm64 case).  This should be handled automatically I believe.

Reply:
Hmm, it wasn't.. I was getting a crash here in Unity, and had to add this here to fix.

Reply:
Perhaps that was an older version before the `__sig` attribute was added?    If you look at the generated code for emscripten_webgl_do_create_context you should see that it has auto-generated conversion code that is generated by `convertPointerParams` in `src/jsifier.js`.